src/Protocol/Diaspora.php

```diff
@@ -4065,6 +4067,7 @@ public static function sendProfile($uid, $recips = false)
 
 		foreach ($recips as $recip) {
 			logger("Send updated profile data for user ".$uid." to contact ".$recip["id"], LOGGER_DEBUG);
+			/// @fixme $profile isn't available here
 			self::buildAndTransmit($profile, $recip, "profile", $message, false, "", true);
```

Comment:
It has to be the ```$owner``` that is fetched - for example - here: https://github.com/friendica/friendica/blob/develop/src/Worker/Delivery.php#L143-L155

Perhaps we should unify this query and use it here.